app/domain/authentication/util/fetch_authenticator_secrets.rb

```diff
@@ -5,11 +5,11 @@ module Authentication
   module AuthnOidc
```

Comment:
If we are making it more general, then let's remove the AuthnOidc module

Reply:
nice catch